packages/bot-api/src/MessageHandler.ts

```diff
@@ -96,31 +96,53 @@ export abstract class MessageHandler {
     }
   }
 
-  async sendCall(conversationId: string, content: CallingContent): Promise<void> {
+  /**
+   * @param userIds Only send message to specified user IDs or to certain clients of specified user IDs
+   */
+  async sendCall(
+    conversationId: string,
+    content: CallingContent,
+    userIds?: string[] | Record<string, string[]>,
+  ): Promise<void> {
     if (this.account?.service) {
       const callPayload = this.account.service.conversation.messageBuilder.createCall(conversationId, content);
-      await this.account.service.conversation.send(callPayload);
+      await this.account.service.conversation.send(callPayload, userIds);
     }
   }
 
-  async sendPoll(conversationId: string, text: string, buttons: string[]): Promise<void> {
+  /**
+   * @param userIds Only send message to specified user IDs or to certain clients of specified user IDs
+   */
+  async sendPoll(
+    conversationId: string,
+    text: string,
+    buttons: string[],
+    userIds?: string[] | Record<string, string[]>,
+  ): Promise<void> {
     if (this.account?.service) {
       const message = this.account.service.conversation.messageBuilder
         .createComposite(conversationId)
         .addText(Text.create({content: text}));
       buttons.forEach(button => message.addButton(button));
-      await this.account.service.conversation.send(message.build());
+      await this.account.service.conversation.send(message.build(), userIds);
     }
   }
 
-  async sendConfirmation(conversationId: string, firstMessageId: string): Promise<void> {
+  /**
+   * @param userIds Only send message to specified user IDs or to certain clients of specified user IDs
+   */
+  async sendConfirmation(
+    conversationId: string,
+    firstMessageId: string,
+    userIds?: string[] | Record<string, string[]>,
```

Comment:
Do we have a type for thus user / client map of `Record<string, string[]` to make clear which string is a user ID and which string is a client ID?

Reply:
Not yet but let me create one!